snow/engine/snowman/bootstrap/config.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 type Config struct {
+	ShouldHalt func() bool
```

Comment:
nit: should we add this at the bottom of the struct so the embedded `AllGetsServer` is at the top?

Reply:
Done